### PR TITLE
Fix attribute typo in Kang

### PR DESCRIPTION
**Pull request by Phyco-Ninja**

### CODE REVIEW

**Comment by rking32** [comment]
@Phyco-Ninja can you add this to gmute too

File: userge/plugins/fun/kang.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ async def kang_(message: Message):
             else:
                 emoji = args[0]
         packname = f"a{user.id}_by_userge_{pack}"
-        custom_packnick = Config.CUSTOM_PACK_NAME or f"@{user.id}'s kang pack"
+        custom_packnick = Config.CUSTOM_PACK_NAME or f"@{user.username}'s kang pack"
```
Comment (rking32) [inline userge/plugins/fun/kang.py:73]: what if `username = None`
what about `{user.username or user.first_name or user.id}`

Comment (Phyco-Ninja) [reply userge/plugins/fun/kang.py:73]: if username = None
it will be ```@None's kang Pack ```

Comment (Phyco-Ninja) [reply userge/plugins/fun/kang.py:73]: Umm btw you don't check if [input is really a emoji or not?](https://github.com/Phyco-Ninja/Userge/blob/patch-1/userge/plugins/fun/kang.py#L71)

Comment (rking32) [reply userge/plugins/fun/kang.py:73]: ok, then add it

**Comment by rking32** [comment]
@Phyco-Ninja now its great 🤘

**Comment by rking32** [comment]
@Phyco-Ninja 
https://github.com/UsergeTeam/Userge/blob/02011cb1b72e0839e89547eab9001a2793bf94c9/userge/plugins/fun/kang.py#L148

**Pull request merged**